client/src/App.css

```diff
@@ -1,33 +1,119 @@
 .App {
-  text-align: center;
+	text-align: center;
+	background-color: #f4f4f4;
 }
 
 .App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+	animation: App-logo-spin infinite 20s linear;
+	height: 134px;
+	pointer-events: none;
 }
 
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+	background-color: #eee;
+	min-height: 240px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	font-size: calc(10px + 2vmin);
+	color: #282c34;
+	-webkit-box-shadow: 0px 2px 15px -8px rgba(0, 0, 0, 0.42);
+	-moz-box-shadow: 0px 2px 15px -8px rgba(0, 0, 0, 0.42);
+	box-shadow: 0px 2px 15px -8px rgba(0, 0, 0, 0.42);
 }
 
-.App-link {
-  color: #61dafb;
+.navbar {
+	width: 90%;
+}
+.topbar {
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+	align-items: center;
 }
 
 @keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+	from {
+		transform: rotate(0deg);
+	}
+	to {
+		transform: rotate(360deg);
+	}
+}
+
+.dark-mode__toggle {
+	background: #ddd;
+	border-radius: 50px;
+	border: 1px solid black;
+	height: 20px;
+	position: relative;
+	width: 40px;
+}
+
+.toggle {
+	background: #61dafb;
+	border-radius: 50px;
+	height: 18px;
+	left: 0;
+	position: absolute;
+	transition: 0.2s;
+	width: 20px;
+}
+
+.toggled {
+	left: 18px;
+}
+
+.Players-list {
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(350px, 1fr));
```

Comment:
The grid looked great for this! You're doing great finding great use cases for grid.